catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -201,6 +201,13 @@
             default=""
             required="true"/>
 
+        <AD id="maximumUploadSize"
+            name="Maximum Endpoint Upload Size"
+            description="The maximum size (in bytes) to allow per client when receiving a POST/PATCH/PUT. Note: This does not affect product upload size, just the maximum size allowed for calls from Intrigue"
+            type="Integer"
```

Comment:
I assume this will be a signed integer, which might limit the upload size to 2.something billion bytes (2GB). I feel like there are situations where someone might want that to be a higher limit

Reply:
It's also worth noting that `Request.getContentLength` returns an int anyway. If we want to guard against the exception in Spark, though, it looks like there's `Request.getContentLengthLong` instead.